be/src/vec/functions/math.cpp

```diff
@@ -350,6 +350,70 @@ struct PowName {
 };
 using FunctionPow = FunctionBinaryArithmetic<PowImpl, PowName, false>;
 
+class FunctionNormalCdf : public IFunction {
+public:
+    static constexpr auto name = "normal_cdf";
+    String get_name() const override { return name; }
+    static FunctionPtr create() { return std::make_shared<FunctionNormalCdf>(); }
+    DataTypePtr get_return_type_impl(const DataTypes& arguments) const override {
+        return make_nullable(std::make_shared<DataTypeFloat64>());
+    }
+    DataTypes get_variadic_argument_types_impl() const override {
+        return {std::make_shared<DataTypeFloat64>(), std::make_shared<DataTypeFloat64>(),
+                std::make_shared<DataTypeFloat64>()};
+    }
+    size_t get_number_of_arguments() const override { return 3; }
```

Comment:
better add blank line between functions/variables

Reply:
i will  add it